core/crypto/src/util.rs

```diff
@@ -92,6 +94,25 @@ impl<
     }
 }
 
+impl PublicKey {
```

Comment:
would `impl From<&AccountId> for PublicKey` be cleaner maybe?

Reply:
I think method does a better job at being self-describing here. Not to mention, not all `AccountId`s can be converted this way.